modules/olm-building-operator-catalog-image.adoc

```diff
@@ -16,10 +16,15 @@ Cluster administrators can build a custom Operator catalog image based on the Pa
 
 For this example, the procedure assumes use of a mirror registry that has access to both your network and the Internet.
 
+[NOTE]
+====
+Only the Linux version of the `oc` client can be used for this procedure, because the Windows and macOS versions do not provide the `oc adm catalog build` command. 
+====
+
 .Prerequisites
 
 * Workstation with unrestricted network access
-* `oc` version 4.3.5+
+* `oc` version 4.3.5+ (Linux only)
```

Comment:
Could this also work as:

* Linux only: `oc` version 4.3.5+

Reply:
Went with "`oc` version 4.3.5+ Linux client"